tests/test_event.py

```diff
@@ -152,3 +157,306 @@ def test_json_to_xml_to_json_nesting(self):
         # The type changed from int to str but that's not really avoidable
         assert json_event.data['candy'][0]['apple'] == '2'
         assert json_event.data['candy'][1]['grape'] == '3'
+
+
+parser = etree.XMLParser(remove_blank_text=True)
+#used to ignore spacing differences and look at basic XML structure
+def xml_formatter(xml_str):
+    return etree.tostring(etree.XML(xml_str, parser=parser))
+
+#used to ignore spacing differences and look at basic JSON structure
+def json_formatter(json_str):
+    return json.dumps(json.loads(json_str))
+
+
+@contextmanager
+def throws_excpetion(*exceptions):
```

Comment:
Is there an advantage to using this vs using https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertRaises?

Reply:
Nope, in fact I'm going to switch over to using that instead.  Didn't realize we already had that available.